central/rbac/utils/common.go

```diff
@@ -81,3 +83,19 @@ func fetchRoles(ctx context.Context, roleStore datastore.DataStore, roleIDs set.
 	}
 	return roles
 }
+
+func convertRoleBindingSearchResult(r search.Result) (string, bool) {
+	var (
+		roleID      string
+		clusterRole bool
+	)
+	roleIDMatches := r.Matches["k8srolebinding.role_id"]
+	if len(roleIDMatches) == 1 {
+		roleID = roleIDMatches[0]
+	}
+	clusterRoleMatches := r.Matches["k8srolebinding.cluster_role"]
+	if len(clusterRoleMatches) == 1 {
+		clusterRole, _ = strconv.ParseBool(clusterRoleMatches[0])
+	}
+	return roleID, clusterRole
```

Comment:
recommend using option map. If the field is moved, this will fail because the path is hard coded.

Reply:
Hmm, I looked at [the option map](https://github.com/stackrox/stackrox/blob/master/central/rbac/k8srolebinding/mappings/map.go#L10) before and it renders to `k8s_role_binding.X` instead.

Seems like it's only used in the service within the `ListSubjects` API, should we just change the option map to use the "correct" `k8srolebinding` instead or is this a breaking change?

Reply:
I am not following you here. I did a quick test with the options map you linked above and following was my observation:
```
fmt.Println(OptionsMap.MustGet(search.ClusterRole.String()))

&{k8s_role_binding.cluster_role SEARCH_BOOL false false ROLEBINDINGS }
```

Reply:
Use the options map attached to the schema, `schema.RoleBindingsSchema.OptionsMap`. With that you can remove the `map.go` because it is obsolete.